static/js/tippyjs.js

```diff
@@ -174,6 +175,22 @@ export function initialize() {
         e.currentTarget._tippy.destroy();
     });
 
+    delegate("body", {
+        target: "#message_type_icon",
```

Comment:
I might call this ID `#compose_message_type_icon`, to better signal its purpose when reading this code, like here, that doesn't clearly indicate what part of the app it relates to.